src/Components/Pages/PortfolioSize/PortfolioSize.scss

```diff
@@ -64,3 +64,29 @@
     }
   }
 }
+
+@media print {
+  #portfolio-size-page {
+    .headline-section__address__print {
+      padding-bottom: 0.75rem; //12px
+    }
+
+    .headline-section__back-link {
+      display: none;
+    }
+
+    .content-box {
+      border: unset;
+      .accordion__chevron {
+        display: none;
+      }
+      .content-box__footer {
+        display: none;
+      }
+    }
+
+    .divider__print {
+      border-bottom: 1px solid black !important;
+    }
```

Comment:
I think these pills might be too light to read. 
![image](https://github.com/user-attachments/assets/44a2d3b4-872c-498c-aa7d-da4625224fd3)

```suggestion
    }
    .acris-links__pill {
      color: black
    }
```

Reply:
good call!